### PR TITLE
fix: Classify not found policies and improve errors

### PR DESCRIPTION
**Pull request by erezrokah**
Replaces https://github.com/cloudquery/cloudquery/pull/684

### CODE REVIEW

**Review by disq — COMMENTED** [review]
I would use `%q` when referring to user inputs.

File: pkg/ui/console/policy.go [hunk 1]
```diff
@@ -43,15 +44,22 @@ func FilterPolicies(policyPath string, policies policy.Policies) (policy.Policie
...
 		return nil, err
 	}
 	if found {
 		return policy.Policies{p}, nil
 	}
-	return nil, fmt.Errorf("no policy with name %s found in configuration or remote. Available in config: %s", policyName, policies.All())
+
+	configPolicies := policies.All()
+	if len(configPolicies) == 0 {
+		return nil, diag.FromError(fmt.Errorf("no valid policy with name %s found. If using a local policy directory, ensure the path is correct and the directory exists", policyName), diag.USER)
```
Comment (disq) [inline pkg/ui/console/policy.go:58]: I would use `%q` when referring to user inputs. Line 61 also.

Comment (erezrokah) [reply pkg/ui/console/policy.go]: Thanks for the tip! Done in https://github.com/cloudquery/cloudquery/pull/697/commits/7353ee0fc13cc7165471437df302f0fff230927a (I didn't wrap the array as it wraps each individual element and doesn't look that nice).

Comment (disq) [reply pkg/ui/console/policy.go]: ty, array isn't user input anyway.

**Review by disq — APPROVED** [review]

**Pull request merged**